src/form-question-types.rst

```diff
@@ -1658,33 +1659,151 @@ In the parameters column, write ``max-pixels=`` followed by the desired maximum
+
+When built-in audio recording is enabled and recording is initiated, a recording control bar appears at at the top of the screen. At the top left of this bar is an icon to represent whether recording is currently ongoing or paused (1). To the right of this icon is the current length of the recording (2).
+
+.. warning::
+
+  Pause is only available on Android 7.0 and above. On lower Android versions, the pause button is hidden.
+
+At the right of the control bar are a pause button (3) and a stop button (4). When the pause button is tapped, recording is temporarily suspended and the button icon changes to a microphone. When the microphone is tapped, recording is resumed. Recording can be paused and resumed as many times as desired. When the stop button is tapped, the recording is ended and can no longer be modified.
+
+Recording status is also displayed below the audio question text. There is a time representing the current length of the recording (5) and a diagram (6) representing the volume of the recording over time. The diagram provides confirmation that the microphone is working and can help a user ensure an even, sufficient volume.
+
+Other questions can be included on the same screen as a built-in recording question. As shown in the screenshot above, this makes it possible to capture quantitative content while recording. To achieve this, put the questions in a :ref:`field list <field-list>`.
+
+During recording, the user is prevented from leaving the current question screen. However, it is safe to use other applications or to lock the device screen.
+
+Once recording is stopped, the control bar disappears. The recording is made available for playback below the question text.
+
+To replace the audio captured, first delete the current file and then record again.
+
+In some rare cases such as the device running out of space, the recording may complete successfully but not be attached to the form. If this happens, a dialog will be displayed explaining that the file is available but needs to be accessed manually. You can find these files in the ``recordings`` folder of the :ref:`Collect directory <collect-directory>`. This folder is never cleared so consider emptying it yourself once you have retrieved its files.
+
+.. _customizing-audio-quality:
+
+Customizing audio quality
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 1.29
+
+  `ODK Collect v1.29.0 <https://github.com/getodk/collect/releases/tag/v1.29.0>`_, `pyxform` v1.3.0, `XLSForm Online` v2.3.0
+
+The quality of audio recordings can be customized using the ``quality`` parameter. If a ``quality`` is specified, the built-in recorder is always used, regardless of Collect settings. If no ``quality`` is specified and Collect is :ref:`configured to use the built-in recorder <use-external-app-for-audio-recording>`, ``normal`` is used. The available quality values are:
```

Comment:
"If a `quality` is specified, the built-in recorder is always used, regardless of Collect settings" isn't strictly true but it does feel like explaining the `external` exception might not make sense seeing as it's documented below. What do you think?

Reply:
Yes, that sounds right to me. In this context it seems like what's most useful.